notebooks/1_intelligent_agents_fall_2021/index.md

```diff
@@ -0,0 +1,135 @@
+<div align="center">
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <h1 style="font-size: 40px; margin: 10px 0;">AI - Intelligent Agents</h1>
+    <h1 style="font-size: 20px; font-weight: 400;">Sharif University of Technology - Computer Engineering Department</h1>
+    <br>
+    <h4 style="font-size: 18px; font-weight: 400; color:#555">Amirreza Mirzaei, Bardia Mohammadi, Sina Elahimanesh</h4>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+</div>
+<hr>
+
+Table of contents
+==============
+- [Introduction](#Introduction)
+- [Intelligent agents](#Intelligent-agents)
+    - [Vacuum world agent example](#Vacuum-world-agent-example)
+- [Rational agents and performance measure](#Rational-agents-and-performance-measure)
+    - [Rationality vs perfection](#Rationality-vs-perfection)
+    - [Autonomy](#Autonomy)
+- [Task environment (PEAS)](#Task-environment-(PEAS))
+    - [Types of environment](#Types-of-environment)
+    - [PEAS example](#PEAS-example)
+- [Type of agents](#Type-of-agents)
+    - [Reflex agents](#Reflex-agents)
+    - [Goal-based agents](#Goal-based-agents)
+    - [Utility-based agents](#Utility-based-agents)
+- [Conclusion](#Conclusion)
+- [References](#References)
+
+# Introduction
```

Comment:
Write out at least a paragraph for this section and try to explain why this topic is important.

Reply:
It's not actually completed yet!